djangoProject/profiles/models.py

```diff
@@ -45,6 +45,16 @@ class BaseProfile(models.Model):
         blank=True,
     )
 
+    logo = models.ImageField(null=True, blank=True)
+
+    user = models.ForeignKey(
+        'users.User',
+        on_delete=models.CASCADE
+    )
+
+    def __str__(self):
+        return self.company_name
+
 
 class StartUpProfile(BaseProfile):
     project_information = models.TextField()
```

Comment:
Define the Investor Model

Action: In your investors app, create an Investor model that includes the necessary fields and a foreign key to User.
https://github.com/Sf-Project-Labs/forum-project-stage/issues/39